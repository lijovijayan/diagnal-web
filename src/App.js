import React from 'react'
import StoreProvider from './store/store.provider'

import './styles/app.styles.scss'
import { Movies } from './containers/movies'
function App() {
    return (
        <StoreProvider>
            <div className="app-container">
                <Movies />
            </div>
        </StoreProvider>
    )
}

export default App
