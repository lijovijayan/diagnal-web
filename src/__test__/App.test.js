import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import App from '../App'

describe('testing whole application', () => {
    it('rendered properly', () => {
        const component = shallow(<App />)
        const tree = toJson(component)
        expect(tree).toMatchSnapshot()
    })
})
