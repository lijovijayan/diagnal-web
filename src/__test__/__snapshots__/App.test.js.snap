// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing whole application rendered properly 1`] = `
<StoreProvider>
  <div
    className="app-container"
  >
    <Movies />
  </div>
</StoreProvider>
`;
